### PR TITLE
fix import error in IPEX Check

### PR DESCRIPTION
**Pull request by y199387**

### CODE REVIEW

File: python/nano/src/bigdl/nano/pytorch/trainer/Trainer.py [hunk 1]
```diff
@@ -99,6 +99,8 @@ def __init__(self, num_processes: int = 1,
                     "Fall back to regular pytorch.")
             use_ipex = False
 
+        self.use_ipex = use_ipex
+
```
Comment (yangw1234) [inline python/nano/src/bigdl/nano/pytorch/trainer/Trainer.py:103]: let's use `self.use_ipex` for rest of this function

File: python/nano/test/pytorch/utils/_train_ipex_callback.py [hunk 2]
```diff
@@ -25,11 +25,10 @@
 
 class CheckIPEXCallback(Callback):
     def on_train_start(self, trainer, pl_module):
+        if trainer.use_ipex == False:
+           warnings.warn("CheckIPEXCallback is used, but ipex fail. ")
```
Comment (yangw1234) [inline python/nano/test/pytorch/utils/_train_ipex_callback.py:29]: "but ipex is disabled"?

File: python/nano/src/bigdl/nano/pytorch/trainer/Trainer.py (hunk 1 above)
Comment (y199387) [reply python/nano/src/bigdl/nano/pytorch/trainer/Trainer.py:103]: Done

File: python/nano/test/pytorch/utils/_train_ipex_callback.py (hunk 2 above)
Comment (y199387) [reply python/nano/test/pytorch/utils/_train_ipex_callback.py]: Done

**Comment by y199387** [comment]
http://10.112.231.51:18888/job/BigDL-NanoTests-PR-Validation/238/

**Comment by yangw1234** [comment]
LGTM

**Pull request merged**